### ISSUE TITLE
Forge connections append info to server name

### ISSUE THREAD

**Issue by itzg** [itzg/mc-router#1]
...such as:

```
time="2019-04-28T21:31:45Z" level=info msg="Got handshake" nextState=1 protocolVersion=404 server="forgennb.my.domain\x00FML2\x00" serverPort=25565
```

**Comment by itzg** [comment]
Fixed in 5875b0d231fa859e9deb03e2b8d68f933d8e2ce9

**Comment by itzg** [comment]
Released in https://github.com/itzg/mc-router/releases/tag/1.3.2

**Comment by itzg** [comment]
Fixed correctly in 96399cfd40ff4ec5c3a2092fe174e14e8385be84 and release in 1.3.3
